### DISCUSSION TITLE
Import & Settings –– JSON: Syntax error, malformed JSON.

### DISCUSSION THREAD

**Discussion by fritzisa** [code4recovery/12-step-meeting-list#724] [Q&A]
When refreshing we started receiving the message above.

Here is one of the two we're having problems refreshing:

https://www.d35pa-aa.org/wp-admin/admin-ajax.php?action=meetings&key=c092a973561757a0f800a6bc5822dd2a

Any help would be appreciated!

HTTP://area59aa.org

**Comment by joshreisner** [comment]
i'm not seeing a malformed JSON error, are you still getting it? if so, maybe take a screenshot?

feed is looking valid to meeting guide: https://meetingguide.org/validate?url=https%3A%2F%2Fwww.d35pa-aa.org%2Fwp-admin%2Fadmin-ajax.php%3Faction%3Dmeetings%26key%3Dc092a973561757a0f800a6bc5822dd2a

**Comment by fritzisa** [comment]
Hi Josh,

Thanks for your reply.

I refreshed District 42 again and this is the result:

Scott Fritzinger  |  Area 59 – Web Servant  |  (610) 762-7792

**Reply by joshreisner** [reply]
i don't see any image but the links still look good to me 👍

**Reply by fritzisa** [reply]
I'm sorry Josh, my previous reply was from my email. I've inserted the screengrab below.
We have (2) District feeds that are producing the same message:

Import & Settings –– JSON: Syntax error, malformed JSON:
https://www.d35pa-aa.org/wp-admin/admin-ajax.php?action=meetings&key=c092a973561757a0f800a6bc5822dd2a
https://padistrict42.org/wp-admin/admin-ajax.php?action=meetings&key=854aec833cc75bb085e64a72b899a2bb

<img width="1382" alt="Screen Shot 2022-03-06 at 4 04 51 PM" src="https://user-images.githubusercontent.com/95769327/158030574-83b2f788-0fbd-4388-99b1-fe8d1f2b47a0.png">

**Reply by joshreisner** [reply]
Thanks @fritzisa -- it's a real puzzle! 🤔 It's working just fine on my development machine, and both URLs look good on the meeting guide validator. Here's what I'm seeing:

<img width="954" alt="Screen Shot 2022-03-12 at 2 11 31 PM" src="https://user-images.githubusercontent.com/1551689/158036823-eb4274ea-a8f1-499b-b360-ce4d7bdf2f58.png">

I imported fine and can click "refresh" fine.

Do you have command line access on your website (the one you're importing from)? If so, try running `curl "https://www.d35pa-aa.org/wp-admin/admin-ajax.php?action=meetings&key=c092a973561757a0f800a6bc5822dd2a"` and see if you see json.

The only thing I can think of is if there's some connection problem between your two websites. It's extra strange since you can import other feeds without incident.

**Comment by gkovats** [comment]
Hi @fritzisa, we released an update that _may_ fix this issue. At the https://www.d35pa-aa.org site, you'd have to work with them on these steps. They'll need to:
- Update the 12-step-meeting-list plugin
- Test this url to see if it shows the meeting feed:
https://www.d35pa-aa.org?rest_route=/tsml/meetings&key=c092a973561757a0f800a6bc5822dd2a
- Update the area site to use this new feed url

Hopefully this produces a new result.
